click_option_group/_core.py

```diff
@@ -259,6 +268,14 @@ def handle_parse_result(self, option: GroupedOption, ctx: click.Context, opts: d
 
             raise click.UsageError(error_text, ctx=ctx)
 
+    def get_help_record(self, ctx: click.Context) -> ty.Tuple[str, str]:
+        if all(o.hidden for o in self.get_options(ctx).values()):
```

Comment:
Maybe we could move this check to `handle_parse_result` method?

Reply:
done